ONBOARDING.md

```diff
@@ -19,6 +19,7 @@ invalid and are superseded by the TSC Charter.
 * Invite them to the HackerOne [Node.js team](https://hackerone.com/nodejs/team_members).
   Have them turn on two-factor authentication after they have joined.
 * Add them to the `tsc` and `crypto-export` mailing lists.
+* Add them to the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group as owners, so they receive Calendar invites for TSC meetings.
```

Comment:
Should I be able to access this link? I get a "content unavailable" error message.

Reply:
Ah never mind. Just saw the invite and the link works after accepting.